.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing to Http Methods
```

Comment:
What do you think of using `HTTP` as is used in the module names?

Reply:
Makes sense. I'll update that.